tests/templates/test_embeddings.py

```diff
@@ -499,15 +499,6 @@ class TestQAOAEmbedding:
               (3, (1, 6), [qml.RX, qml.RX, qml.RX, qml.MultiRZ, qml.MultiRZ, qml.MultiRZ,
                    qml.RY, qml.RY, qml.RY, qml.RX, qml.RX, qml.RX])]
 
-    @pytest.mark.parametrize('n_wires, weight_shape, expected_queue', QUEUES)
-    def test_queue(self, n_wires, weight_shape, expected_queue):
-        """Checks the queue for the default settings."""
-
-        with qml.tape.OperationRecorder() as rec:
-            QAOAEmbedding(features=list(range(n_wires)), weights=np.zeros(shape=weight_shape), wires=range(n_wires))
-
-        for gate, expected_gate in zip(rec.queue, expected_queue):
-            assert isinstance(gate, expected_gate)
```

Comment:
Removed because the recorder now records the gate, which would have to manually decomposed. This test is replaced by the new decomposition tests anyways.